docs/3.tutorials/fts/0-intro.md

```diff
@@ -0,0 +1,59 @@
+---
+id: introduction
+title: Fungible Tokens Zero to Hero
+sidebar_label: Introduction
+---
+
+> In this _Zero to Hero_ series, you'll find a set of tutorials that will cover every aspect of a fungible token (FT) smart contract.
+> You'll start by interacting with a pre-deployed contract and by the end you'll have built a fully-fledged FT smart contract that supports every extension of the standards.
+
+---
+
+## Prerequisites
+
+To complete these tutorials successfully, you'll need:
+
+- [Rust](/develop/prerequisites)
+- [A NEAR Wallet](https://wiki.near.org/getting-started/creating-a-near-wallet)
+- [NEAR-CLI](/tools/near-cli#setup)
+
+:::info New to Rust?
+If you are new to Rust and want to dive into smart contract development, our [Quick-start guide](/develop/quickstart-guide) is a great place to start.
+:::
+
+---
+
+## Overview
+
+These are the steps that will bring you from **_Zero_** to **_Hero_** in no time! 💪
+
+| Step | Name                                                                         | Description                                                                          |
+| ---- | ---------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
+| 1    | [Pre-deployed contract](/tutorials/fts/predeployed-contract) | Receive FTs without the need to code, create, or deploy a smart contract.            |
+| 2    | [Contract architecture](/tutorials/fts/skeleton)             | Learn the basic architecture of the FT smart contract and compile the code.             |
+| 3    | [Defining a Token](/tutorials/fts/minting)                            | Flesh out what it means to have a FT and how you can customize your own          |
+| 4    | [Circulating Supply](/tutorials/fts/upgrade-contract)        | Learn how to create an initial supply and have the token show up in your wallet.                          |
+| 5    | [Registering Accounts](/tutorials/fts/enumeration)                    | Explore how you can implement and understand the storage management standard to avoid malicious users from draining your funds.  |
```

Comment:
```suggestion
| 5    | [Registering Accounts](/tutorials/fts/register-accounts)                    | Explore how you can implement and understand the storage management standard to avoid malicious users from draining your funds.  |
```